File: java/memento/TextEditorState.java
package memento;

public class TextEditorState {

    private String content;

    public TextEditorState(String content) {
        this.content = content;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }
}

File: PHP/Patterns/src/Behavioral/Memento/TextEditorState.php
<?php

namespace Behavioral\Memento;

class TextEditorState
{
    private $content;

    

    public function __construct(string $content)
    {
        $this->content = $content;
    }

    /**
     * Get the value of content
     */ 
    public function getContent()
    {
        return $this->content;
    }

    /**
     * Set the value of content
     *
     * @return  self
     */ 
    public function setContent($content)
    {
        return $this->content = $content;

    }
}